src/fa/replayserver.py

```diff
@@ -74,7 +74,7 @@ def readDatas(self):
             #This prefix means "P"osting replay in the livereplay protocol of FA, this needs to be stripped from the local file            
             if data.startsWith("P/"):
                 rest = data.indexOf("\x00") + 1
-                self.__logger.info("Stripping prefix '" + str(data.left(rest)) + "' from replay.")
+                self.__logger.info("Stripping prefix '" + str(data.left(rest) - 1) + "' from replay.")
```

Comment:
It's later appended to another QByteArray - I don't know if we want to convert it to Python string and back just for some nice syntax.